lib/client-assets.php

```diff
@@ -230,6 +230,15 @@ function gutenberg_register_vendor_scripts() {
 		'promise',
 		'https://unpkg.com/promise-polyfill/promise' . $suffix . '.js'
 	);
+
+	// TODO: This is only necessary so long as WordPress 4.9 is not yet stable,
+	// since we depend on the newly-introduced wp-api-request script handle.
+	//
+	// See: gutenberg_ensure_wp_api_request (compat.php).
+	gutenberg_register_vendor_script(
+		'wp-api-request-shim',
+		'https://rawgit.com/WordPress/wordpress-develop/master/src/wp-includes/js/api-request.js'
```

Comment:
@aduth, even though this is a temporary measure, shouldn't we prefer a `cdn.rawgit.com`-type URL?

Per their homepage:

> [cdn.rawgit.com] No traffic limits or throttling. Files are served via StackPath's super fast global CDN.

vs.

> [rawgit.com] Excessive traffic will be throttled and blacklisted.

Reply:
Now that you mention it, I'd meant to look if we need to use rawgit.com at all, or if we can just reference the GitHub.com link directly. The latter is blocked for script tags in the browser, but since this occurs server-side, we might be able to reference the raw JS file straight from the source.

Reply:
Also worth noting that we're not sending much traffic here. The plugin includes a pre-downloaded copy, and in development the script is downloaded and cached for 24 hours.

Reply:
Fair enough. :)